src/NuGet.Services.Metadata.Catalog.Monitoring/Providers/PackageTimestampMetadataResourceV2Provider.cs

```diff
@@ -7,15 +7,21 @@
 using Microsoft.Extensions.Logging;
 using NuGet.Protocol;
 using NuGet.Protocol.Core.Types;
+using NuGet.Services.Metadata.Catalog.Helpers;
 
 namespace NuGet.Services.Metadata.Catalog.Monitoring
 {
     public class PackageTimestampMetadataResourceV2Provider : ResourceProvider
     {
-        public PackageTimestampMetadataResourceV2Provider(ILoggerFactory loggerFactory)
+        private readonly IGalleryDatabaseQueryService _galleryDatabaseQueryService;
```

Comment:
>_galleryDatabaseQueryService [](start = 54, length = 28)

This name is a little verbose. What do you think of a name like `_galleryDatabase`? If you change this name, please also update `PackageTimestampMetadataResource`.

Reply:
Addressed with https://github.com/NuGet/NuGet.Services.Metadata/pull/537/commits/efccb258c8c58307490c364713a284d0fa12322e